includes/abstracts/abstract-wc-stripe-payment-gateway.php

```
@@ -35,6 +35,70 @@ abstract class WC_Stripe_Payment_Gateway extends WC_Payment_Gateway_CC {
+	 * The cache is not written here on purpose: the option-change hooks
+	 * registered by the main gateway invalidate it, so the next read reloads the
+	 * persisted value regardless of which write path ran.
+	 *
+	 * @since 10.9.0
+	 *
+	 * @param array $settings The settings to persist.
+	 * @return bool Whether the option was actually written (matches `update_option`).
+	 */
+	public function update_settings( array $settings ): bool {
+		return WC_Stripe::write_settings_option( $settings );
+	}
```

PHPMD (2.15.0)

[error] The method update_settings is not named in camelCase. (undefined)

(CamelCaseMethodName)